css/properties/line-break.json

```diff
@@ -157,24 +155,22 @@
             ],
             "support": {
               "chrome": {
-                "version_added": "≤83"
+                "version_added": "25"
```

Comment:
In Chrome 25, `-webkit-line-break: auto` and `-webkit-line-break: loose` look the same (in fact all values look the same), which makes me think it isn't actually supported in Chrome 1+.